targets/xorg

```diff
@@ -64,11 +87,17 @@ if [ "${ARCH#arm}" = "$ARCH" ]; then
     fi
 
     # Unhold xserver-xorg-video-intel, modification will be reapplied later.
-    apt-mark unhold "$inteldriver" 2>/dev/null || true
+    if [ "${DISTROAKA:-"$DISTRO"}" = 'debian' ]; then
+        apt-mark unhold "$inteldriver" 2>/dev/null || true
```

Comment:
what are you going to do about the binary hack on the intel driver?  In this case, you could probably also make an overlay patch...